main/webapp/modules/core/langs/translation-fr.json

```diff
@@ -1,35 +1,35 @@
 {
     "name": "Français",
     "core-index/help": "Aide",
-    "core-index/contributors": "Contributeurs :",
+    "core-index/contributors": "Contributeurs ",
     "core-index/new-proj-name": "Nom du nouveau projet :",
     "core-index/download-now": "Télécharger $1 maintenant.",
     "core-index/delete-key": "Supprimer la préférence",
-    "core-index/id": "ID du projet :",
-    "core-index/subject": "Sujet :",
+    "core-index/id": "ID du projet ",
```

Comment:
And here.

Reply:
Removed the trailing spaces and updated the tests.